app/models/groupe_instructeur.rb

```diff
@@ -89,11 +88,30 @@ def can_delete?
   def routing_to_configure?
     rule = routing_rule
     return true if !(rule.is_a?(Logic::Eq) && rule.left.is_a?(Logic::ChampValue) && rule.right.is_a?(Logic::Constant))
-    !routing_rule_matches_tdc?
+    return true if !routing_rule_matches_tdc?
+    non_unic_rule?
+  end
+
+  def non_unic_rule?
+    return false if routing_rule.nil?
+    return false if routing_rule.right == empty
+    routing_rule.in?(other_groupe_instructeurs.map(&:routing_rule))
+  end
```

Comment:
Voir le commentaire plus haut : si on désolidarise `routing_to_configure?` et `non_unic_rule?` dans les templates haml, on peut retirer la ligne 92 `no_unic_rule?` de la méthode `routing_to_configure?` 

et réécrire 

```suggestion
  def non_unic_rule?
    return false if routing_to_configure?
    routing_rule.in?(other_groupe_instructeurs.map(&:routing_rule))
  end
```

de manière à exclure tous les cas ou la règle est mal configurée.